pkg/client/didexchange/client.go

```diff
@@ -48,17 +52,25 @@ func New(ctx provider) (*Client, error) {
 	if err != nil {
 		return nil, err
 	}
+
 	didexchangeSvc, ok := svc.(dispatcher.DIDCommService)
 	if !ok {
 		return nil, errors.New("cast service to DIDExchange Service failed")
 	}
+
+	store, err := ctx.StorageProvider().GetStoreHandle()
+	if err != nil {
+		return nil, err
+	}
+
 	c := &Client{
 		didexchangeSvc:           didexchangeSvc,
 		wallet:                   ctx.CryptoWallet(),
 		inboundTransportEndpoint: ctx.InboundTransportEndpoint(),
 		// TODO channel size - https://github.com/hyperledger/aries-framework-go/issues/246
 		actionCh: make(chan dispatcher.DIDCommAction, 10),
 		msgCh:    make(chan dispatcher.StateMsg, 10),
+		record:   persistence.NewConnectionRecord(store),
```

Comment:
"Record" makes it sound like this is an individual record rather than a handle to storage for a particular type.

Reply:
Will rename it to `Recorder`

Reply:
done